log_parser.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo -n "{}" > filter_logs.json
+declare -a filters=("status" "retention_schedule" "litigation_hold" "contact_phone" "language" "referred" "hate_crime" "servicemember" "intake_format" "location_state" "primary_complaint" "contact_first_name" "contact_last_name" "contact_email" "violation_summary" "location_name" "location_name_1" "location_name_2" "location_city_town" "create_date_start" "create_date_end" "summary" "origination_utm_campaign" "assigned_to" "dj_number" "tags" "public_id" "primary_statute" "district" "reported_reason" "commercial_or_public_place")
+jq -r '.[].link' exportedLogRecords.json | while read link ; do
+    linkparams=( $(echo ${link} | tr "?" "\n") )
+    for j in $(echo ${linkparams[2]} | tr "&" "\n"); do
```

Comment:
consider renaming `j` and `k` semantically, to make it more clear what content they should have